### PR TITLE
Release/2.16

### PR DESCRIPTION
**Pull request by nenadV91**
# Release 2.16.0

DO NOT SQUASH!!

# Notable changes:

- Handle Native EthFlow orders https://github.com/cowprotocol/explorer/pull/257
- Hide Fee row from order details for not filled limit-orders https://github.com/cowprotocol/explorer/pull/261
- Display order type (limit|market|liquidity) https://github.com/cowprotocol/explorer/pull/259
- Fix order type tooltip text https://github.com/cowprotocol/explorer/pull/264

### CODE REVIEW

File: src/components/orders/DetailsTable/index.tsx [hunk 1]
```diff
@@ -82,7 +82,19 @@ const tooltip = {
...
+  type: (
+    <div>
+      CoW Protocol supports three type of orders – market, limit and liquidity:
+      <ul>
+        <li>A market order is an order to buy or sell at the market&apos;s current best available price</li>
+        <li>A limit order is an order to buy or sell at an arbitrary price specified by the user</li>
+        <li>A liquidity order is an order that market makers can provide as a source of liquidity</li>
+      </ul>
+      In addition, an order can be &quot;fill or kill&quot; or &quot;partially fillable&quot;. Currently all order types
+      can be &quot;fill or kill&quot; – only market orders can be either &quot;fill or kill&quot; or &quot;partially
```
Comment (alfetopito) [inline src/components/orders/DetailsTable/index.tsx:94]: This is inaccurate:

```suggestion
      can be &quot;fill or kill&quot; – but only liquidity orders can be either &quot;fill or kill&quot; or &quot;partially
```

**Review by alfetopito — APPROVED** [review]
One minor issue with the tooltip

File: src/components/orders/DetailsTable/index.tsx (hunk 1 above)
Comment (anxolin) [reply src/components/orders/DetailsTable/index.tsx:94]: I flagged this first in slack, and then in the doc I made a proposal https://docs.google.com/document/d/1au3_QDBkzo4CgW9TlmHcQWMEnsQUaH7m0UItgnFEke0/edit#

image


double check with @MindyCoW / @avsavsavs   pls

**Review by anxolin — APPROVED** [review]
Soft approve. Pls review the order  types

**Comment by elena-zh** [comment]
Changes LGTM besides issues reported above.

**Comment by anxolin** [comment]
BTW here is also the comment in the PR unaddressed https://github.com/cowprotocol/explorer/pull/259#discussion_r1032391071

**Comment by anxolin** [comment]
@nenadV91  @MindyCoW validated the text proposal above. Green light!

**Pull request merged**